packages/adminrouter/extra/src/lib/auth/open.lua

```diff
@@ -159,7 +159,7 @@ function _M.init(use_auth)
     end
 
     -- /package/
-    res.access_package_endpoint = function()
+    res.access_package_endpoint = function(package_path)
```

Comment:
Do we need to pass `package_path` argument if we're not using it later?

Reply:
I'm not sure, but this is consistent with https://github.com/dcos/dcos/blob/82abd9676231e24c8a888e3e8c2b65df82d800e0/packages/adminrouter/extra/src/lib/auth/open.lua#L101